flexget/plugins/input/my_anime_list.py

```diff
@@ -58,67 +59,41 @@ class MyAnimeList(object):
     @cached('my_anime_list', persist='2 hours')
     def on_task_input(self, task, config):
         entries = []
-        parameters = {'u': config['username'], 'status': 'all', 'type': 'anime'}
-        selected_status = config.get('status', list(STATUS.values()))
-        selected_types = config.get('type', list(ANIME_TYPE.values()))
+        selected_status = config['status']
+        selected_types = config['type']
 
         if not isinstance(selected_status, list):
             selected_status = [selected_status]
 
         if not isinstance(selected_types, list):
             selected_types = [selected_types]
 
+        for index, value in enumerate(selected_status):
+            selected_status[index] = STATUS[value]
```

Comment:
`selected_status = [STATUS[s] for s in selected_status]` would still be more idiomatic than these two lines.

Other than that: LGTM.